test/lib/spender_web/controllers/page_controller_test.exs

```diff
@@ -5,7 +5,7 @@ defmodule SpenderWeb.PageControllerTest do
   describe "Page Controller" do
      test "renders the homepage on the root url", %{conn: conn} do
       conn = get conn, "/"
-      assert html_response(conn, 200)
+      assert html_response(conn, 200) =~ "Welcome to Mula API."
```

Comment:
should be "Welcome to the Mula API"

Reply:
so i shouldn't include the `=~` ?

Reply:
@br4inii  you should just the text in the page should be `Welcome to the Mula API` not `Welcome to Mula API` there is a word missing in the second one :)